cfgov/ask_cfpb/documents.py

```diff
@@ -1,36 +1,8 @@
 from django_elasticsearch_dsl import Document, fields
 from django_elasticsearch_dsl.registries import registry
```

Comment:
Should these changes be in a separate PR for ask_cfpb?

Reply:
Agree. I created a separate ask_cfpb PR: https://github.com/cfpb/consumerfinance.gov/pull/6076